src/actions/BreachActions.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
for the breach actions we want to open the breach modal for a specific breach for a location and breaches for a batch and a specific breach for a batch. In this PR only the `setFridgeBreach` action is fully implemented but can see the others work basically the same.